lib/server.go

```diff
@@ -1232,16 +1226,6 @@ func (srv *Server) _handleGetSnapshot(pp *Peer, msg *MsgDeSoGetSnapshot) {
 func (srv *Server) _handleSnapshot(pp *Peer, msg *MsgDeSoSnapshotData) {
 	srv.timer.End("Get Snapshot")
 
-	// If the node is running a Fast-HotStuff validator and the consensus is running,
```

Comment:
The FastHotStuffConsensus can only be initialized after Hypersync has completed. No reason to add this check here. If the node isn't syncing snapshots, then this function will exit early anyway. That is already handled below in line 1256